index.html

```diff
@@ -1283,6 +1285,12 @@ <h4>
             connecting</a> and then follow the steps to <a>establish a connection
             with the remote playback device</a>.
           </p>
+          <div class="note">
+            A user agent that implements browser initiated remote playback
+            should consider how it interacts with other browser policies that
+            affect media playback, such as <code><a>autoplay</a></code> policies
```

Comment:
I would avoid linking to HTML's autoplay. If you want to link to something maybe https://html.spec.whatwg.org/multipage/media.html#allowed-to-play would be closer to "autoplay policies"?

Reply:
Updated the text to refer to that definition.

Reply:
I can't see the update

Reply:
@mounirlamouri  Sorry, forgot to add commit.  PTAL